frontend/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "emeq-counter-frontend",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "npx parcel index.html",
+    "build": "npx parcel build index.html",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "engines": {
+    "node": "12.13.0",
+    "npm": "6.12.0"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {},
+  "devDependencies": {
+    "elm-hot": "^1.1.4",
```

Comment:
Why we don't add `elm` to devDependencies?

Reply:
Parcel handles elm build step with 'elm-hot' and 'node-elm-compiler' AFAIK. So we don't need to add elm itself to dependencies.

Reply:
Thank you for this detail.